adding post about unlocking the blog (#4283)
unlock-protocol/unlock: unlock-protocol.com/blog/unlock-this-blog.md

---
title: Unlock this blog
subTitle:
authorName: Julien Genestoux
publishDate: July 24, 2019
description:
image:
---

> Unlock is a [protocol for memberships](/blog/protocol-for-membership). It helps creators monetize their community (fans, supporters, or even simple users) by helping them deploy a lock and sell access keys to this lock (they can chose their currency, price, the duration of the keys and the number of keys they want to sell...).

Reviewer: I think that we may want to rephrase this a bit ` It helps creators monetize their community` as the language implies they are monetizing access to the audience as opposed to the content/creation/access etc

We decided to create a lock for members of our community. It's a good way for us to show what's possible with a lock. Since our goal was not to make money, we decided to charge _0 eth_ for the keys.

When integrating the lock on this blog, we decided to reflect the membership status in 3 different things:

1. The lower bar at the bottom of every page. ⬇️ Since I am a member, mine shows this:

![member bar](/static/images/blog/unlock-blog/member-bar.png)

> 2. On each post, users can view and leave [comments](#comments), but only if they are members themselves.

Reviewer: ```suggestion
2. On each post, __members__ can view and leave [comments](#comments).
```

> 3. On some posts, we have decided to lock some content. For example, read this Google Slideshow from a presentation I made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).

Reviewer: ```suggestion
3. We've locked some exclusive content. For example, this Google Slideshow from a presentation made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).
```

# Bundling

One thing to note is that the lock we use on this blog is in no way limited to this site. We could (and will, stay tuned...) re-use the lock on different sites or applications. As a matter of facts, you could even add *our* lock to *your* site or application, if you wanted to only grant access to people who are part of our community. That approach is how creators can deploy "bundles": a single lock is used on several different websites or applications!

# Several locks

> Another important detail is that we could very well add multiple locks to this site. Maybe we'd have one of our most loyal fans (with a limit on the number of keys), and maybe we'd have one for our everyone else, using another currency or one which we would "share" with several other applications to be part of their bundle... etc.

Reviewer 2: maybe worth clarifying that you only need one key to either of those locks to access the content?

Author: Good point! Added

authorName: Julien Genestoux
publishDate: July 24, 2019
description: We've put a lock on our own blog! Members can leave comments, read exclusive content and will receive a Non Fungible Token.
image: /static/images/blog/unlock-blog/hero.jpg
---

Unlock is a [protocol for memberships](/blog/protocol-for-membership). It helps creators monetize by selling access keys to their community -fans, supporters, or even simple users- by helping them deploy a lock and sell access keys to this lock (they can chose their currency, price, the duration of the keys and the number of keys they want to sell...).

We decided to create a lock for members of our community. It's a good way for us to show what's possible with a lock. Since our goal was not to make money, we decided to charge _0 eth_ for the keys.

When integrating the lock on this blog, we decided to reflect the membership status in 3 different things:

1. The lower bar at the bottom of every page. ⬇️ Since I am a member, mine shows this:

![member bar](/static/images/blog/unlock-blog/member-bar.png)

2. On each post, __members__ can view and leave [comments](#comments).

> 3. We've locked some exclusive content. For example, this Google Slideshow from a presentation made at Betaworks about [Non Fungible Tokens](/blog/non-fungible-tokens-betaworks).

Author: This would be an extremely ling link :/
